[FR] Users with unique titles (#4980)
ppy/osu-wiki: wiki/People/Users_with_unique_titles/fr.md

- ![][flag_JP] [iceOC](https://osu.ppy.sh/users/5482401)
- ![][flag_JP] [kiyozi11](https://osu.ppy.sh/users/9551196)
- ![][flag_JP] [Seren58](https://osu.ppy.sh/users/15252950)
- ![][flag_JP] [uone](https://osu.ppy.sh/users/5321719)
 
Gagnants de la [CWC 2020](/wiki/Tournaments/CWC/2020) avec le titre utilisateur **osu!catch Champion**:
 
- ![][flag_KR] [Motion](https://osu.ppy.sh/users/3885626)
- ![][flag_KR] [qwhj1027](https://osu.ppy.sh/users/7547506)
- ![][flag_KR] [R-18](https://osu.ppy.sh/users/4637369)
- ![][flag_KR] [Rocma](https://osu.ppy.sh/users/566276)
- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)
- ![][flag_KR] [T s u m i](https://osu.ppy.sh/users/4080520)
 
Gagnants de la [MWC 4K 2020](/wiki/Tournaments/MWC/2020_4K) avec le titre utilisateur **osu!mania Champion**:
 
- ![][flag_BR] [andreymc](https://osu.ppy.sh/users/5691061)
- ![][flag_BR] [Guilhermeziat](https://osu.ppy.sh/users/3661387)
- ![][flag_BR] [Kyut](https://osu.ppy.sh/users/9328824)
- ![][flag_BR] [Lothus](https://osu.ppy.sh/users/9530019)
- ![][flag_BR] [Remu](https://osu.ppy.sh/users/8700468)
- ![][flag_BR] [SillyFangirl](https://osu.ppy.sh/users/2288363)
 

> ### Gagnants de compétition de mapping officielles

Reviewer: ```suggestion
### Gagnants de compétitions de mapping officielles
```

| ![][flag_FR] [Kurokotei](https://osu.ppy.sh/users/398275) | [Kurokotei](https://osu.ppy.sh/beatmaps/artists/58) |
| ![][flag_JP] [nakanojojo](https://osu.ppy.sh/users/10631848) | [Nakanojojo](https://osu.ppy.sh/beatmaps/artists/25) |
| ![][flag_JP] [Namirin](https://osu.ppy.sh/users/2264828) | [\*namirin](https://osu.ppy.sh/beatmaps/artists/18) |
| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | [nekodex](https://osu.ppy.sh/beatmaps/artists/1) |
| ![][flag_US] [onumi](https://osu.ppy.sh/users/11204867) | [onumi](https://osu.ppy.sh/beatmaps/artists/32) |
| ![][flag_PH] [rtnario](https://osu.ppy.sh/users/16222702) | [LEAF XCEED Music Division](https://osu.ppy.sh/beatmaps/artists/88) |
| ![][flag_US] [skymuted](https://osu.ppy.sh/users/7734050) | [Fractal Dreamers](https://osu.ppy.sh/beatmaps/artists/15) |
| ![][flag_ES] [sotuiofficial](https://osu.ppy.sh/users/14779258) | [Rising Sun Traxx](https://osu.ppy.sh/beatmaps/artists/41) |
| ![][flag_RU] [tieff](https://osu.ppy.sh/users/89619) | [tieff](https://osu.ppy.sh/beatmaps/artists/34) |
| ![][flag_CA] [VINXIS](https://osu.ppy.sh/users/4323406) | [VINXIS](https://osu.ppy.sh/beatmaps/artists/22) |
 

> ### Gagnants de compétition de design des mascottes

Reviewer: ```suggestion
### Gagnants de compétitions de design des mascottes
```

# Utilisateurs ayant des titres uniques
 

> Les titres utilisateurs sont généralement associés avec des [groupes d'utilisateurs](/wiki/People/The_Team#user-groups), mais les utilisateurs listés ci-dessous ont reçu leurs titres pour d'autres raisons. La plupart de ces titres sont des récompenses pour des contributions ou des succès, le reste étant des preuves de l'histoire iconique ou drôle d'osu!.

Reviewer: ```suggestion
Les titres utilisateurs sont généralement associés avec des [groupes d'utilisateurs](/wiki/People/The_Team#user-groups), mais les utilisateurs listés ci-dessous ont reçu leurs titres pour d'autres raisons. La plupart de ces titres sont des récompenses pour des contributions remarquables ou l'accomplissement de succès uniques, le reste étant des preuves de l'histoire iconique d'osu!.
```

- ![][flag_CA] [ktgster](https://osu.ppy.sh/users/53378)
- ![][flag_NL] [Kyshiro](https://osu.ppy.sh/users/640611)
- ![][flag_NL] [Lesjuh](https://osu.ppy.sh/users/44308)
- ![][flag_TW] [Licnect](https://osu.ppy.sh/users/352804)
- ![][flag_AU] [Lilac](https://osu.ppy.sh/users/58197)
- ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023)
- ![][flag_AU] [m980](https://osu.ppy.sh/users/3288)
- ![][flag_US] [MMzz](https://osu.ppy.sh/users/128993)
- ![][flag_CN] [Necho](https://osu.ppy.sh/users/4086593)
- ![][flag_FR] [Nofool](https://osu.ppy.sh/users/672430)
- ![][flag_FI] [PianoLuigi](https://osu.ppy.sh/users/9665915)
- ![][flag_US] [RLC](https://osu.ppy.sh/users/1047883)
- ![][flag_MY] [RyoKazuka](https://osu.ppy.sh/users/6258586)
- ![][flag_CN] [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057)
- ![][flag_LY] [Soul Evans](https://osu.ppy.sh/users/4490770)
- ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340)
- ![][flag_RU] [xbopost](https://osu.ppy.sh/users/6842421)
- ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661)
- ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768)
 
### Nominateurs d'élite
 

> Depuis 2020, l'activité des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) et des membres de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team)  dans le support du système de ranking a été récompensée avec le titre **Elite Nominator**. Ce titre est donné chaque début d'année pour les contributions faites pendant la dernière, et est enlevé lorsque l'utilisateur quitte la team.

Reviewer: ```suggestion
Depuis 2020, la contribution des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) et des membres de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) dans le système de ranking a été récompensée avec le titre **Elite Nominator**. Ce titre est attribué en début d'année pour les contributions apportées durant l'année précédente et est retiré lorsque l'utilisateur quitte l'équipe.
```

 
- ![][flag_TW] [bossandy](https://osu.ppy.sh/users/360437)
- ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410)
- ![][flag_PH] [Jemzuu](https://osu.ppy.sh/users/7890134)
- ![][flag_DE] [Mordred](https://osu.ppy.sh/users/7265097)
 
### Gagnants de compétitions de mapping Aspire
 

> La compétition de mapping Aspire possède une variation spéciale du titre Elite Mapper appelée **Elite Mapper : Aspirant** en tant que prix.

Reviewer: ```suggestion
La compétition de mapping Aspire possède une variation spéciale du titre Elite Mapper appelée **Elite Mapper : Aspirant** en tant que prix pour la première place.
```

 
- ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000)
- ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699)
- ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438)
- ![][flag_NL] [Mr HeliX](https://osu.ppy.sh/users/2330619)
- ![][flag_FI] [ProfessionalBox](https://osu.ppy.sh/users/3250792)
- ![][flag_TW] [SnowNiNo_](https://osu.ppy.sh/users/2506267)
- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)
- ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231)
 
 

> ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) a reçu ce titre en ayant déjà le titre Elite Mapper, il a donc été "amélioré" vers **Elite Mapper II : Aspirant**. ![][flag_KR][Spectator](https://osu.ppy.sh/users/702598) utilise actuellement le titre **osu!catch Champion** a la place.

Reviewer: ```suggestion
![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) a reçu ce titre en ayant déjà le titre Elite Mapper, il a donc été "amélioré" vers **Elite Mapper II : Aspirant**. ![][flag_KR][Spectator](https://osu.ppy.sh/users/702598) utilise actuellement le titre **osu!catch Champion** à la place.
```

 
### Gagnant de compétition osu!remix

> La compétition osu!remix attribue le titre **osu!mixer** en tant que premier prix.

Reviewer: ```suggestion
La compétition osu!remix attribue le titre **osu!mixer** en tant que prix pour la première place.
```

| ![][flag_JP] [nakanojojo](https://osu.ppy.sh/users/10631848) | [Nakanojojo](https://osu.ppy.sh/beatmaps/artists/25) |
| ![][flag_JP] [Namirin](https://osu.ppy.sh/users/2264828) | [\*namirin](https://osu.ppy.sh/beatmaps/artists/18) |
| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | [nekodex](https://osu.ppy.sh/beatmaps/artists/1) |
| ![][flag_US] [onumi](https://osu.ppy.sh/users/11204867) | [onumi](https://osu.ppy.sh/beatmaps/artists/32) |
| ![][flag_PH] [rtnario](https://osu.ppy.sh/users/16222702) | [LEAF XCEED Music Division](https://osu.ppy.sh/beatmaps/artists/88) |
| ![][flag_US] [skymuted](https://osu.ppy.sh/users/7734050) | [Fractal Dreamers](https://osu.ppy.sh/beatmaps/artists/15) |
| ![][flag_ES] [sotuiofficial](https://osu.ppy.sh/users/14779258) | [Rising Sun Traxx](https://osu.ppy.sh/beatmaps/artists/41) |
| ![][flag_RU] [tieff](https://osu.ppy.sh/users/89619) | [tieff](https://osu.ppy.sh/beatmaps/artists/34) |
| ![][flag_CA] [VINXIS](https://osu.ppy.sh/users/4323406) | [VINXIS](https://osu.ppy.sh/beatmaps/artists/22) |
 
### Gagnants de compétition de design des mascottes
 

> La osu!team a tenu des compétitions d'art pour que les membres de la communauté aient une chance de dessiner ou redessiner certaines mascottes du jeu. Les gagnants de ces compétitions, ainsi que les personnes pour qui les dessins ont finalement été choisis comme officiels, ont reçu le titre **osu!artist**.

Reviewer: ```suggestion
L'osu!team a tenu des compétitions d'art permettant aux membres de la communauté de dessiner ou redessiner certaines mascottes du jeu. Les gagnants de ces compétitions, ainsi que les personnes pour qui les dessins ont finalement été choisis comme officiels ont reçu le titre **osu!artist**.
```

 
- ![][flag_US] [Crowie](https://osu.ppy.sh/users/6894067) a dessiné **Mocha** et a aidé à produire beaucoup d'autres de ses artworks.
- ![][flag_ID] [JMC](https://osu.ppy.sh/users/774010) a gagné la compétition [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2).
- ![][flag_US] [Thievley](https://osu.ppy.sh/users/4717672) a re-dessiné **Yuzu**.
- ![][flag_ID] [xiemon](https://osu.ppy.sh/users/5203667) a dessiné **Mani and Mari**.
 
### Hall of Famers 
<!--TODO: traduire ce truc -->

> Ces trois utilisateurs ont été récompensés du titre **osu! Hall of Famer** pour avoir joué au jeu au niveau le plus haut, et contribué lorsque la communauté était encore jeune.

Reviewer: ```suggestion
Ces trois utilisateurs ont été récompensés du titre **osu! Hall of Famer** pour avoir joué au jeu au niveau le plus haut, et énormément contribué lorsque la communauté était naissante.
```

<!-- TODO: reasons -->
 
- ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589)
- ![][flag_PH] [James](https://osu.ppy.sh/users/5728)
- ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781)
 
### Storyboarders d'élite
 
Les storyboarders ayant démontré un talent exceptionnel et impressionné la communauté au sens large ont reçu le titre **Elite Storyboarder**.
- ![][flag_US] [11t](https://osu.ppy.sh/users/2112092)
- ![][flag_FR] [Damnae](https://osu.ppy.sh/users/989377)
- ![][flag_PL] [Exile-](https://osu.ppy.sh/users/2559349)
 
### Relatif au gameplay

> Ces joueurs ont gagné leurs titres à travers soit des accomplissements fous, ou la complétion d'un défi spécifique.

Reviewer: ```suggestion
Ces joueurs ont gagnés leurs titres à travers des prouesses exceptionnelles, ou la complétion d'un défi spécifique.
```

> | Utilisateur | Titre | Accomplissement |

Reviewer: ```suggestion
| Utilisateur | Titre | Prouesse |
```

| :-- | :-- | :-- |
| ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | A réalisé le premier FC nomod sur [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |

> | ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | A FC toutes les maps osu!taiko classées avant le 3 novembre 2019 |

Reviewer: ```suggestion
| ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | A complété à 100% toutes les beatmaps du mode osu!taiko classées avant le 3 novembre 2019 |
```

| ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | *Non-Documenté* <!-- TODO: a reference to TWEWY's 104 building, because he got to rank #104 in a certain amt of time.. need to investigate further --> |

> | ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | A joué toutes les maps osu!standard classée avant le 27 avril 2019 |

Reviewer: ```suggestion
| ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | A joué toutes les beatmaps du mode osu!standard classée avant le 27 avril 2019 |
```
